lib/ansible/modules/net_tools/netbox/netbox_device.py

```diff
@@ -198,13 +205,18 @@ def main():
         validate_certs=dict(type="bool", default=True)
     )
 
+    global module
```

Comment:
Please refrain from using `global`.

Reply:
It's not ideal indeed and when refactoring Netbox modules into a NetboxModule class, we'll get rid of it. But for now I think it's good enough. That is how it's done in all of those modules:
```
lib/ansible/modules/identity/onepassword_facts.py
lib/ansible/modules/web_infrastructure/nginx_status_facts.py
lib/ansible/modules/web_infrastructure/jira.py
lib/ansible/modules/files/file.py
lib/ansible/modules/system/openwrt_init.py
lib/ansible/modules/system/parted.py
lib/ansible/modules/net_tools/ipify_facts.py
lib/ansible/modules/monitoring/spectrum_device.py
lib/ansible/modules/packaging/language/pip_package_info.py
lib/ansible/modules/packaging/os/package_facts.py
lib/ansible/modules/storage/glusterfs/gluster_volume.py
lib/ansible/modules/storage/glusterfs/gluster_heal_facts.py
lib/ansible/modules/network/cloudengine/ce_netstream_global.py
lib/ansible/modules/network/cloudengine/ce_vxlan_global.py
lib/ansible/modules/network/cloudengine/ce_info_center_global.py
lib/ansible/modules/network/cloudengine/ce_evpn_global.py
lib/ansible/modules/network/cloudengine/ce_bfd_global.py
lib/ansible/modules/cloud/misc/terraform.py
lib/ansible/modules/cloud/misc/rhevm.py
lib/ansible/modules/cloud/memset/memset_dns_reload.py
lib/ansible/modules/cloud/memset/memset_zone_domain.py
lib/ansible/modules/cloud/memset/memset_server_facts.py
lib/ansible/modules/cloud/memset/memset_zone_record.py
lib/ansible/modules/cloud/memset/memset_zone.py
lib/ansible/modules/cloud/memset/memset_memstore_facts.py
lib/ansible/modules/cloud/cloudstack/cs_facts.py
lib/ansible/modules/cloud/amazon/ec2_asg.py
lib/ansible/modules/cloud/amazon/ec2_instance.py
```
What do you think? Is it okay for now?

Reply:
I understand that these modules have used `global` and really its not ideal and I would recommend to have that removed from the code. Please make sure that this gets removed during the refactor. Thanks!